pandas/core/strings/accessor.py

```diff
@@ -1,12 +1,12 @@
 from __future__ import annotations
 
 import codecs
+from collections.abc import Callable  # noqa: PDF001
 from functools import wraps
 import re
 from typing import (
     TYPE_CHECKING,
     Hashable,
-    Pattern,
```

Comment:
we should not be using Pattern from typing https://www.python.org/dev/peps/pep-0585/